### PR TITLE
First Update of README.md file

### PR DESCRIPTION
**Pull request by jlcarrascof**
🏅 Features Implemented

We implemented the following requirements:

    ✅ Created the first version of our professional README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,16 +121,22 @@ Browser (Chrome recommended)
...
 
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-No future features
+Here will write some future features
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (ahmidou09) [inline README.md:142]: please add the link of the page of the issue

File: README.md [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+    Apply JavaScript best practices and language style guides in code.
+    Use AAA pattern for unit tests.
+    Write units tests for a JavaScript app.
+    Follow Gitflow.
+    Solve simple git conflicts.
+    Send and receive data from an API.
+    Use API documentation.
+    Understand and use JSON.
+    Make JavaScript code asynchronous.
+    Perform a code review for a team member.
```
Comment (ahmidou09) [inline README.md:21]: please can you remove this part and add the description of the projects

File: README.md [hunk 3]
```diff
@@ -107,16 +121,22 @@ Browser (Chrome recommended)
...
+
 👤 Javier J. Martínez F.
 
 - GitHub: [@jlcarrascof]
 - Twitter: [@javiersistemas]
 
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🔭 Future Features <a name="future-features"></a>
```
Comment (ahmidou09) [inline README.md:138]: please can you add future-features

**Review by ahmidou09 — CHANGES_REQUESTED** [review]
Hi @jlcarrascof ,
Good job so far!
But there are some issues that you still need to work on to go to the next project but you are almost there!
a

Highlights!

✔️Very professional README

Status: Required Changes ♻️⛔️

Check the comments under the review.

File: README.md (hunk 2 above)
Comment (ahmidou09) [inline README.md:21]: please can you remove this part and add the description of the projects

File: README.md [hunk 4]
```diff
@@ -49,13 +61,15 @@
 
 ### Key Features <a name="key-features"></a>
```
Comment (ahmidou09) [inline README.md:63]: please can you add future-features

**Comment by jlcarrascof** [comment]
Thanks for your feedback @ahmidou09, I'll work on it. 🙂👍

**Comment by jlcarrascof** [comment]
I **fixed the description of the project** and I put the **features in progress and in to-do status**.

Thanks,

Javier.-

**Review by ahmidou09 — APPROVED** [review]
Hi @jlcarrascof 
b

Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

**Pull request merged**